changelog/fix-4422-set-available-method-to-card-after-disabling-upe

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Update id card to available payment method after disabling UPE
```

Comment:
What do you think about something like "Hide UPE payment methods after disabling UPE" to make the changelog entry a little clearer as to what's been fixed?